### ISSUE TITLE
parameters not being resolved for OAS3 'content' type parameters

### ISSUE THREAD

**Issue by codectl** [marshmallow-code/apispec#830]
The OAS3 spec defines the following as a possible parameter:

```yaml
...
parameters:
   - in: path
     name: Pet
     content:
        application/json:
            schema: PetSchema
```

In this example, ```PetSchema``` is not resolved to a ```ref```.

I will create a PR.

**Comment by codectl** [comment]
PR #831 closes this issue.

**Comment by lafrech** [comment]
Fixed in #831.
